site/content/docs/4.3/components/progress.md

```diff
@@ -127,7 +127,7 @@ The striped gradient can also be animated. Add `.progress-bar-animated` to `.pro
   <div class="progress">
     <div class="progress-bar progress-bar-striped" role="progressbar" aria-valuenow="75" aria-valuemin="0" aria-valuemax="100" style="width: 75%"></div>
   </div>
-  <button type="button" class="btn btn-secondary mt-3" data-toggle="button" id="btnToggleAnimatedProgress" aria-pressed="false" autocomplete="off">
```

Comment:
Maybe this change shouldn't be in this PR?

Reply:
Or am I wrong... I thought you had another PR for removing autocomplete.

Reply:
think it makes sense to keep it here as it's related to toggle buttons and how they're then presented/used throughout the docs, rather than leave it in and remove it later (the other branch is still in limbo as it's not quite clear if Firefox still has that bug that prompted it in the first place or not)